src/authn/authn.ts

```diff
@@ -315,9 +315,9 @@ async function loadIndex (
 ): Promise<AuthenticationContext> {
   // Loading preferences is more than loading profile
   try {
-    ;(await isPublic)
+    context = await (isPublic
```

Comment:
Why overwrite `context` which was passed in from https://github.com/solid/solid-ui/blob/master/src/authn/authn.ts#L312?
And also from https://github.com/solid/solid-ui/blob/master/src/authn/authn.ts#L314 it seems that `isPublic` is a boolean?

I'll dig into this function later today, adding unit tests, code documentation, and a usage example, so we can all understand better what the desired behaviour is here. Without that, this is very hard for me to review. Will review the rest of this PR independently first.

Reply:
Moving this to https://github.com/solid/solid-ui/issues/209